build.assets/grpcbox.mk

```diff
@@ -14,9 +14,14 @@ else
 GRPCBOX ?= $(GRPCBOX_BASE_NAME):$(BUILDBOX_VERSION)
 endif
 
+# UID and GID are used to run the grpcbox as the current user.
+UID := $$(id -u)
+GID := $$(id -g)
```

Comment:
Inline the `id` calls below? We don't need the Makefile variables.

Reply:
+1, these variables leak to the outer Makefile, right?